### PR TITLE
Add script checking case when  HMI responds WARNINGS(success:true) to…

### PR DESCRIPTION
**Pull request by vvvakulenko**
… UI.PerformAudioPassThru and ANY successfull result code to TTS.Speak
Script is a part of TS #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova , Please review

File: test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+config.SDLStoragePath = commonPreconditions:GetPathToSDL() .. "storage/"
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @vvvakulenko, storage path variable is not used. Please remove.

File: test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      ttsChunks = {{ text = "Makeyourchoise", type = "TEXT" }},
+      appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+    })
+    :Do(function(_,data)
+      self.hmiConnection:SendNotification("TTS.Started",{ })
+      local function ttsSpeakResponce()
+        self.hmiConnection:SendResponse (data.id, data.method, resultCodes[i], {})
+        self.hmiConnection:SendNotification("TTS.Stopped")
+      end
+    end)
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @vvvakulenko, the function should be call.
And in scope of TTS.Speak will be good to be called after 1500msec.
You can use:
RUN_AFTER(ttsSpeakResponce,1500)

File: test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        {{fieldName = "audioPassThruDisplayText1", fieldText = "DisplayText1"},
+        {fieldName = "audioPassThruDisplayText2", fieldText = "DisplayText2"}},
+      maxDuration = 2000,
+      muteAudio = true
+    })
+    :Do(function(_,data)
+      local function UIPerformAoudioResponce()
+        self.hmiConnection:SendResponse(data.id, "UI.PerformAudioPassThru", "WARNINGS", {})
+      end
+      commonTestCases:DelayedExp(1500)
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @vvvakulenko, the function should be call.
And in scope of UI.PerformAudioPassThru will be good to be called after 1500msec.
You can use:
RUN_AFTER(UIPerformAoudioResponce,1500)

Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @vvvakulenko, this  commonTestCases:DelayedExp(1500) is not needed.

File: test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua (hunk 1 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @istoimenova , updated in commit 17ed7dc07e0cff8d970b028361fc2bd7e3e34701

File: test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua (hunk 2 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @istoimenova , updated in commit 17ed7dc07e0cff8d970b028361fc2bd7e3e34701

File: test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua (hunk 3 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @istoimenova , updated in commit 17ed7dc07e0cff8d970b028361fc2bd7e3e34701

Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/013_ATF_PerformAudioPassThru_UI_PerformAudioPassThru_WARNINGS.lua]: @istoimenova , updated in commit 17ed7dc07e0cff8d970b028361fc2bd7e3e34701

**Comment by vvvakulenko** [comment]
@istoimenova , Please see updates commit 17ed7dc07e0cff8d970b028361fc2bd7e3e34701

**Pull request merged**